tuna/miopen/fin_class.py

```diff
@@ -78,10 +79,17 @@ def __init__(self, **kwargs):
     self.all_configs = []
     self.fin_list = []
     self.multiproc = False
+    self.pending = []
+    self.first_pass = True
 
     self.__dict__.update(
         (key, value) for key, value in kwargs.items() if key in allowed_keys)
 
+    self.config_type = ConfigType.convolution if self.config_type is None else self.config_type
```

Comment:
An unset config type should be an error if the higher layers will populate it ?

Reply:
config_type is populated by FinClass, reference to config_type was removed from WorkerInterface. Default config type is convolution.